dbms/src/Flash/Mpp/TrackedMppDataPacket.h

```diff
@@ -93,7 +88,7 @@ struct MemTrackerWrapper
     {
         int bak_size = size;
         freeAll();
-        memory_tracker = getSharedPtrOfMemTracker(new_memory_tracker);
+        memory_tracker = new_memory_tracker;
```

Comment:
maybe we can add `likely` in `MemTrackerWrapper::alloc` and `MemTrackerWrapper::free`

Reply:
Done